layouts/default.vue

```diff
@@ -1,17 +1,22 @@
 <template>
 	<div id="defaultContainer">
 		<Header />
-		<nuxt />
+		<main>
```

Comment:
@TeddTech In my comment I said:

> I agree with @cindyli's point about using a `<div>` for now and leaving a TODO comment.

I also said:

> Semantically, I think that you're correct in placing the `<aside>` outside of the `<main>` tag as it is a complementary landmark…

However, you used `<main>` instead of a `<div>` and you put the `<aside>` inside the `<main>`. This is not semantically correct, and as per @cindyli's comment (https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/134#issuecomment-608451020), this doesn't address some of these layout issues.